hezar/trainers/trainer.py

```diff
@@ -511,11 +511,7 @@ def push_to_hub(
             commit_message=commit_message,
         )
         self.train_dataset.config.push_to_hub(
-            repo_id,
```

Comment:
@HamedBabaei jan, what exactly is causing this behavior? Shouldn't all parameters be on one line each?

Reply:
@arxyzan Nice hint! The way that you put this before was right and in the middle of the way we configured it to consider multiple lines for parameters, so this change wasn't reverted back somehow, so I reverted back manually and runner pre-commit to make sure it works! 

Thanks for pointing this out!